draggablepanel/src/main/java/com/github/pedrovgs/DraggableView.java

```diff
@@ -323,14 +326,24 @@ public boolean isClosed() {
    * @param ev captured.
    * @return true if the view is going to process the touch event or false if not.
    */
-  @Override public boolean onInterceptTouchEvent(MotionEvent ev) {
+  @Override public boolean onInterceptTouchEvent(@NonNull MotionEvent ev) {
```

Comment:
This annotation has no sense. You are overriding this method from the Android SDK and is the Android SDK who is going to call this method. The contract specified in the onInterceptTouchEvent declaration says MotinEvent can't be null. Please, remove this annotation and the import.